src/routes/Onboarding/CreateAccount.tsx

```diff
@@ -105,14 +106,12 @@ const CreateAccount: FC<CreateAccountProps> = ({
         <Button
           variant="primary"
           isDisabled={checkChanges()}
-          as={Link}
-          to={ROUTES.ACCOUNTS}
           size="large"
           w={desktopMode ? undefined : '100%'}
           onClick={() => {
-            createAccount(accountName, phrase as MnemonicPhraseType).then(() =>
-              onCreate()
-            )
+            createAccount(accountName, phrase as MnemonicPhraseType)
+              .then(() => onCreate())
+              .finally(() => navigate(ROUTES.ACCOUNTS))
```

Comment:
To my mind this step must be in `onCreate` callback.

Reply:
CreateAccount also used in App where useNavagate hook can not be used simply.

Reply:
If navigate is no longer required for account creation it can be omitted, as its also not required for create account modal